x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_dashboard.tsx

```diff
@@ -229,8 +235,8 @@ export const ComplianceDashboard = () => {
   }, [
```

Comment:
there's no need to have this `selectInitialTab()` function, you can just remove it

```js
  useEffect(() => {
      const cspmTotalFindings = getCspmDashboardData.data?.stats.totalFindings;
      const kspmTotalFindings = getKspmDashboardData.data?.stats.totalFindings;
      const installedPolicyTemplatesCspm = getSetupStatus?.cspm?.status;
      const installedPolicyTemplatesKspm = getSetupStatus?.kspm?.status;
      let preferredDashboard = CSPM_POLICY_TEMPLATE;

      // cspm has findings
      if (!!cspmTotalFindings) {
        preferredDashboard = CSPM_POLICY_TEMPLATE;
      }
      // kspm has findings
      else if (!!kspmTotalFindings) {
        preferredDashboard = KSPM_POLICY_TEMPLATE;
      }
      // cspm is installed
      else if (
        installedPolicyTemplatesCspm !== 'unprivileged' &&
        installedPolicyTemplatesCspm !== 'not-installed'
      ) {
        preferredDashboard = CSPM_POLICY_TEMPLATE;
      }
      // kspm is installed
      else if (
        installedPolicyTemplatesKspm !== 'unprivileged' &&
        installedPolicyTemplatesKspm !== 'not-installed'
      ) {
        preferredDashboard = KSPM_POLICY_TEMPLATE;
      }
      setSelectedTab(preferredDashboard);
  }, [
   getCspmDashboardData.data?.stats.totalFindings,
    getKspmDashboardData.data?.stats.totalFindings,
    getSetupStatus?.cspm?.status,
    getSetupStatus?.kspm?.status,
  ]);
```